### PR TITLE
feat: added login page template + all needed CSS/HTML components

### PR DESCRIPTION
**Pull request by YourivHDenHaag**
Work in progress: adding CTA Card (for login pages)

### CODE REVIEW

File: components/Header/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.denhaag-header {
+  border-bottom: var(--denhaag-header-border-bottom);
```
Comment (Yolijn) [inline components/Header/src/index.scss:2]: If we use `--denhaag-header-border-bottom-color`, `--denhaag-header-border-bottom-width` etc. and a hardcoded `border-bottom-style: solid;` we will be able to use the same key/value pairs in Figma as this doesn't support the same composite tokens. The same goes for a hypothetical other output such as for iOS or Android. :)

File: components/Header/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  &__container {
```
Comment (Yolijn) [inline components/Header/src/index.scss:4]: Seems odd to call the content of a container (the header in this case) container. Could we maybe call it `__content`?

File: components/Page/src/index.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
+.denhaag-page {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```
Comment (Yolijn) [inline components/Page/src/index.scss:3]: Can we use `grid-template-columns: repeat(12, 1fr);` instead?

File: components/Page/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  grid-auto-flow: column;
+
+  &-header {
+    grid-column: 1 / 13;
+  }
+
+  /* Den Haag XXS */
+  @media (max-width: 359px) {
```
Comment (Yolijn) [inline components/Page/src/index.scss:11]: Lets use CSS variables for all of it, based on the Design Tokens in Proprietary

```css
/* From the proprietary/components/denhaag/page.tokens.json file */
.denhaag-theme {
    --denhaag-page-grid-column-gap: 8px;
    --denhaag-page-grid-margin-inline-start: 24px;
    --denhaag-page-grid-margin-inline-end: 24px;
    --denhaag-page-xs-grid-column-gap: 8px;
    --denhaag-page-xs-grid-margin-inline-start: 24px;
    --denhaag-page-xs-grid-margin-inline-end: 24px;
}

.denhaag-page {
     grid-column-gap: var(--denhaag-page-grid-column-gap);
     margin-inline-start: var(--denhaag-page-margin-inline-start);
     margin-inline-end: var(--denhaag-page-margin-inline-end);
    
    @media (min-width: 360px) {
        --denhaag-page-grid-column-gap: var(--denhaag-page-xs-grid-column-gap, var(--denhaag-page-grid-column-gap));
        --denhaag-page-grid-margin-inline-start: var(--denhaag-page-grid-xs-margin-inline-start, var(--denhaag-page-grid-margin-inline-start));
        --denhaag-page-grid-margin-inline-end: var(--denhaag-page-grid-margin-inline-end, var(--denhaag-page-grid-margin-inline-end));
    }

    @media (min-width: 768px) {
        ...
    }
}
```

File: .storybook/stories/templates/Login-page.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import Breadcrumb from "/components/Breadcrumb/src";
+import { AuthenticationCardGroup } from "/components/CardGroup/src/story-template";
+
+import clsx from "clsx";
+
+import "./story-login.css";
+
+<Meta title="Templates/Login page" />
+
+# Summary page
```
Comment (Yolijn) [inline .storybook/stories/templates/Login-page.stories.mdx:19]: Hmmm is het eigenlijk de Summary page of the Login page?

File: proprietary/Components/src/denhaag/responsive-content.tokens.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+  "denhaag": {
+    "responsive-content": {
+      "max-width": { "value": "calc(100vw - (var(--denhaag-space-inline-xl)))" },
```
Comment (Yolijn) [inline proprietary/Components/src/denhaag/responsive-content.tokens.json:4]: Unfortunately this `calc` is not something that is supported outside of `css` and therefor not a great thing to use in the Style Dictionary design tokens. So we should use the actual value instead :(

**Comment by Yolijn** [comment]
@YourivHDenHaag I think we're almost there. I left two comments and I think mostly the `calc` in style-dictionary should be resolved

File: .storybook/stories/templates/Login-page.stories.mdx (hunk 5 above)
Comment (YourivHDenHaag) [reply .storybook/stories/templates/Login-page.stories.mdx]: Login page, i've changed it.

File: proprietary/Components/src/denhaag/responsive-content.tokens.json (hunk 6 above)
Comment (YourivHDenHaag) [reply proprietary/Components/src/denhaag/responsive-content.tokens.json]: I've removed the calc function and added a border-box alternative with a padding design token.

**Review by Yolijn — APPROVED** [review]

**Pull request merged**